### PR TITLE
#210 Improved ability to send feedback

### PR DESCRIPTION
**Pull request by maik-nack**
- Improved ability to send feedback on the screen "About"

### CODE REVIEW

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -55,8 +55,12 @@
         решение.</string>
     <string name="msg_version">Версия %1$s</string>
     <string name="title_about_activity">О приложении</string>
-    <string name="msg_contact">Если у Вас имеются вопросы,\nпожалуйста,
-        свяжитесь нами:\nnonoconverter@gmail.com</string>
+    <string name="msg_contact">Если у Вас имеются вопросы, предложения и
```
Comment (MaximSmolskiy) [inline app/src/main/res/values-ru/strings.xml]: Probably, "или" is better here

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -52,8 +52,13 @@
         to a nonogram, which has a single logical solution.</string>
     <string name="msg_version">Version %1$s</string>
     <string name="title_about_activity">About</string>
-    <string name="msg_contact">If you have any questions,\nplease contact us:\n
-        nonoconverter@gmail.com</string>
+    <string name="msg_contact">If you have any questions, suggestions and
```
Comment (MaximSmolskiy) [inline app/src/main/res/values/strings.xml]: Probably, "or" is better here

**Review by MaximSmolskiy — APPROVED** [review]
Looks great!

**Review by NikitaSukhanov — APPROVED** [review]
Nice!

**Pull request merged**